backend/controllers/AuthController.ts

```diff
@@ -13,29 +15,71 @@ export class AuthController extends BaseController {
 
   protected initializeRoutes(): void {
     this.router.get('/siwf', this.getSiwf.bind(this));
-    this.router.get('/account', validateAuthToken, this.getAccount.bind(this));
+    this.router.get('/account', this.getAccount.bind(this));
```

Comment:
The current flow of the front-end requires that back-end has the public key to create the authToken. 
However, during the sign-up process, the front-end has to wait on the back-end to send a transaction to the blockchain to create the msaId, so the authToken is not available yet.